seth/tts-CalculatorAssignment/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Calculator</title>
+    <link rel="stylesheet" type="text/css" href="styleKoo.css">
+
+  </head>
+  <body>
+    <h1>TTS - Calculator Assigmnet</h1>
+
+    <div id="tts-ParentCalculator">
+      <div class="rows">
+        <div class="columns" id="Calculator-display">0</div>
+    </div>
+
+    <div class="rows">
```

Comment:
Checkout my html for my calculator and the styling. I would avoid using row divs. Remember to try to avoid unnecessary divs just for the sake of styling. If you can do it much simpler, try that. In this case you really don't need any row container divs you can just `display: grid;` or `display: flex;` on a single container.